assets/css/frontend.less

```diff
@@ -634,6 +634,10 @@
 			&-carousel {
 				direction: ltr;
 
+				.slick-list {
+					max-height: 267px;
```

Comment:
Hard-coding height in px is not ok since there are many different screen sizes and the slider width will vary (so the aspect ratio will be different), please find a way to limit it in % relative to the slider width.

Also, regarding the second issue in the task, it's reported by a few customers so it's there for sure, please ask Andrii how to reproduce it (also make sure that you synced the branch with master).

Reply:
there is now way to limit height in % because slider automatically sets height in pixels when Litespeed cache image lazy loading is enabled
Also, I've tried to set max-height in vh, vw but no luck